docs/shop_administration/catalogs.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Catalogs allow creating filtered product lists for special purposes, for example promos, campaigns, or tenders.
+---
+
+# Catalogs
+
+You can create catalogs which contain a selection of products from the system.
```

Comment:
How about switching the order of these two sentences? First, the reason for using catalogs.